jitpack.yml

```diff
@@ -0,0 +1,4 @@
+jdk:
+  - oraclejdk8
+install:
+   - mvn clean install -DskipTests
```

Comment:
Why skipTests?

Reply:
jitpack just builds the artifacts, tests are run in Travis.